src/App.js

```diff
@@ -1,22 +1,31 @@
 import './App.css';
 import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
 
-function App() {
+function useSemiPersistentState () {
+  const [todoList, setTodoList] = useState(JSON.parse(localStorage.getItem("savedTodoList")));
+
```

Comment:
💯  good job using locale storage here.
But there's a bug here...What do you think will happen if there's nothing saved in the localStorage for the `savedTodoList` key?